### PR TITLE
Admin Guide: Refactoring

### PR DESCRIPTION
**Pull request by nbalacha**
Update the GlusterFs Introduction

Signed-off-by: N Balachandran <nbalacha@redhat.com>

### CODE REVIEW

File: Administrator Guide/GlusterFS Introduction.md [hunk 1]
```diff
@@ -1,63 +1,33 @@
...
-building block of GlusterFS.
-The POSIX compatible GlusterFS servers, use any ondisk file system which supports
-extended attributes (eg: ext4, XFS, etc) to format to store data on disks, can be
-accessed using industry-standard access protocols including Network File System (NFS)
-and Server Message Block (SMB).
+What is Gluster ?
+=================
 
-![640px-glusterfs_architecture](https://cloud.githubusercontent.com/assets/10970993/7412187/dbb94286-ef5d-11e4-93ca-d4959a94db85.png)
+Gluster is a scalable, distributed file system that aggregates disk and memory resources into a single global namespace.
```
Comment (prashanthpai) [inline Administrator Guide/GlusterFS Introduction.md]: The term "memory resources" can be misleading. It's better to change it to `disk storage` or `disk storage capacity` or even just `disks`

File: Administrator Guide/GlusterFS Introduction.md [hunk 2]
```diff
@@ -1,63 +1,33 @@
...
+  * Can use any ondisk filesystem that supports extended attributes
+  * Accessible using industry standard protocols like NFS and SMB
+  * Provides replication, quotas, geo-replication, snapshots and bitrot detection
+  * Allows optimization for different workloads
+  * Open Source
 
-## Commercial offerings and support ##
 
-Several companies offer support or [consulting](http://www.gluster.org/consultants/).
+![640px-glusterfs_architecture](https://cloud.githubusercontent.com/assets/10970993/7412187/dbb94286-ef5d-11e4-93ca-d4959a94db85.png)
```
Comment (prashanthpai) [inline Administrator Guide/GlusterFS Introduction.md]: Can this link be relative instead ?

Comment (nbalacha) [reply Administrator Guide/GlusterFS Introduction.md]: Do you mean copy it into the project?

File: Administrator Guide/GlusterFS Introduction.md (hunk 1 above)
Comment (nbalacha) [reply Administrator Guide/GlusterFS Introduction.md]: Will do.

File: Administrator Guide/GlusterFS Introduction.md (hunk 2 above)
Comment (prashanthpai) [reply Administrator Guide/GlusterFS Introduction.md]: If it's not in the repo, there's nothing to be done here.

Comment (prashanthpai) [reply Administrator Guide/GlusterFS Introduction.md]: You can add it to the repo, provided we know the source and its usage license.

**Pull request merged**